packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -44,10 +44,11 @@
     "@justeat/f-services": "1.7.0"
   },
   "peerDependencies": {
-    "@justeat/browserslist-config-fozzie": ">=1.1.1"
+    "@justeat/browserslist-config-fozzie": ">=1.1.1",
+    "js-cookie": "2.2.1"
```

Comment:
Do we want to mention that consuming apps need to add this when they use this component? In the readme?

Reply:
@kevinrodrigues Will do another PR for this